chains/ethereum/chain.go

```diff
@@ -111,6 +111,10 @@ func InitializeChain(chainCfg *core.ChainConfig, logger log15.Logger, sysErr cha
 	if err != nil {
 		return nil, err
 	}
+	err = conn.LockAndUpdateOpts()
```

Comment:
This call is used in `voteProposal` and `executeProposal` which means that gas prices are updated AT the time at which we want to execute our transactions which makes sense.